docs/tutorials/iers.rst

```diff
@@ -0,0 +1,135 @@
+  compute UT1-UTC, and by default will raise an error if you try to compute
+  UT1-UTC for a time that is outside the bounds of the IERS Bulletin B table
+  (see the `astropy`_ docs on the `UT1/UTC transformation offsets
+  <http://astropy.readthedocs.org/en/latest/time/index.html?highlight=iers#transformation-offsets>`_
+  for more details), like this::
+
+    >>> from astropy.time import Time
+    >>> Time('2040-01-01', scale='utc').ut1    # Convert from UTC to UT1
+    IndexError: (some) times are outside of range covered by IERS table.
+
+* IERS Bulletin A encompasses the observations of recent Earth orientation
+  contained in Bulletin B, while also making extrapolations into the past,
+  before the Bulletin B tables begin, and into the future, after the Bulletin
+  B tables end. The future predictions include leap second additions scheduled
+  to be added.
+
+Let's plot the UT1-UTC from IERS Bulletins A and B to show the difference using
+astropy's IERS machinery:
+
+.. plot::
+
+    import matplotlib.pyplot as plt
+    import numpy as np
+    import astropy.units as u
+    from astropy.time import Time
+
+    # Make the plots pretty
+    import seaborn as sns
+    sns.set(context='talk')
+
+    # Download and cache the IERS Bulletins A and B  using astropy's machinery
+    # (reminder: astroplan has its own function for this: `download_IERS_A`)
+    from astropy.utils.iers import (IERS_A, IERS_A_URL, IERS_B, IERS_B_URL,
+                                    FROM_IERS_B)
+    from astropy.utils.data import download_file
+    iers_a = IERS_A.open(download_file(IERS_A_URL, cache=True))
+    iers_b = IERS_B.open(download_file(IERS_B_URL, cache=True))
+
+    # Get a range of times to plot from 1990-2020
+    time_range = Time("1990-01-01") + np.arange(0, 30, 0.2)*u.year
+
+    # Calculate the difference between UTC and UT1 at those times,
+    # allowing times "outside of the table"
+    DUT1_a, success_a = time_range.get_delta_ut1_utc(return_status=True,
+                                                     iers_table=iers_a)
+    DUT1_b, success_b = time_range.get_delta_ut1_utc(return_status=True,
+                                                     iers_table=iers_b)
+
+    # Compare input times to the times available in the table. For details, see
+    # https://github.com/astropy/astropy/blob/master/astropy/utils/iers/iers.py#L80
+    measurements_from_b = (success_b == FROM_IERS_B)
+
+    # Make a plot of the time difference
+    fig, ax = plt.subplots(figsize=(10,8))
+    ax.axhline(0, color='gray', ls='--', lw=2)
+
+    ax.plot_date(time_range.plot_date,
+                 DUT1_a, '-', label='IERS Bulletin A + extrapolation')
+    ax.plot_date(time_range.plot_date[measurements_from_b],
+                 DUT1_b[measurements_from_b], 'r--', label='IERS Bulletin B')
+
+    ax.set(xlabel='Year', ylabel='UT1-UTC [seconds]')
+    ax.legend(loc='upper right')
+    plt.show()
```

Comment:
I tried this and found it a bit more confusing. For the purposes of this high level example, I think sticking to UTC and UT1, which are the names most likely to show up in warnings, errors and tracebacks, might be sufficient.